lib/run-compilation.js

```diff
@@ -46,7 +46,12 @@ module.exports = function runCompilation (options, compiler, done) {
     }
 
     if (errors.length) {
-      compilation.errors.push(chalk.red(options.formatter(errors)));
+      if (options.emitErrors === false) {
+        var suppressionWarning = 'Stylelint error, reported as warning because emitErrors is set to false';
+        compilation.warnings.push(chalk.yellow(options.formatter(errors + ' - ' + suppressionWarning)));
```

Comment:
So that unexpectedly took me down a rabbit hole :)

Basically, I shouldn't have assumed `errors` to be a simple string. It's actually an array of objects containing details about stylelint errors (and warnings).

Of note is that it also contains a property `severity` with a value of either `warning` or `error`. This will in turn cause the formatter to display the warning with either a ⚠ or a ✖ prefixed, respectively, effectively communicating to the user when they were actually errors.

Thus, I've removed the message again and added two tests that ensure that these icons are actually prefixed when appropriate. Hopefully that's OK with you.

And given that I now understood how the stylelint errors are passed through to Webpack, I actually refactored it a bit to unmark files as containing errors, rather than messing with the stats reporter.

Two things to consider:
- I've used `Object.assign` to keep the `downgradeErrors` function pure. This is only supported in Node 4+, so I've added that to `package.json`, if that's OK.
- The two new tests _are_ dependent on the formatter marking errors and warnings using the mentioned icons. I'm not too happy with that, but I guess it's reasonable to not expect those to change suddenly.